app/Models/Post.php

```diff
@@ -195,7 +195,7 @@ public function toQuasarPayload()
             'northstar_id' => $this->northstar_id,
             'type' => $this->type,
             'action' => $this->action,
-            'quantity' => $this->quantity,
+            'quantity' => $this->quantity ? $this->quantity : $this->signup->quantity,
```

Comment:
I don't think I understand this issue. Why would we ever want to send the signup quantity (which will be the total quantity across all posts) here?

Reply:
@sbsmith86 because as of right now we are not updating the quantity column on the post when a post is submitted